website/blog/2023-10-22-databend-weekly-116.md

```diff
@@ -0,0 +1,113 @@
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTang
+  - name: dantengsky
+  - name: dependabot[bot]
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: flaneur2020
+  - name: JackTan25
+  - name: lichuang
+  - name: PsiACE
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zenus
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### Feature Preview: Managing Background Tasks with SQL
+
+Previously, Databend introduced **Background Service** to allow Databend to run one-off background jobs or cron jobs in daemon mode. This simplifies the complexity of managing data maintenance tasks.
+
+To make it easy to create, manage and maintain background tasks, Databend recently added support for a series of SQL statements including `CREATE TASK`, `ALTER TASK`, `SHOW TASK`, etc.
+
+For example, the following SQL statement instructs Databend to execute a task called `MyTask1` every day at `6am PST` to insert `(1, 2)` and `(3, 4)` into table `t`:
+
+```sql
+CREATE TASK IF NOT EXISTS MyTask1 SCHEDULE = USING CRON '0 6 * * *' 'America/Los_Angeles' COMMENT = 'serverless + cron' AS insert into t (c1, c2) values (1, 2), (3, 4) 
+```
+
+The **Background Service** requires Databend Enterprise Edition. Please [contact the Databend team](https://www.databend.com/contact-us) for upgrade information.
+
+If you are interested in learning more, please check out the resources below:
+
+- [PR #13316 | feat: impl create task sql parser and planner](https://github.com/datafuselabs/databend/pull/13316)
+- [PR #13344 | feat: add SQL syntax support for alter, execute, describe, show, drop task](https://github.com/datafuselabs/databend/pull/13344)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Understanding Configuration Mapping in Databend
+
+For a complex database service like Databend, supporting many configurable options helps developers manage and tune the system. A recently published blog explains the mappings between command line options, environment variables, config files, and code in Databend.
+
+Databend currently supports three configuration methods in decreasing order of priority:
+
+- Command line options are used to override a small number of configs.
+- For K8s clusters, environment variables enable flexibly changing some configs (e.g. feature flags) more elegantly.
```

Comment:
2. Environment Variables: Provide configuration flexibility for Kubernetes clusters.